### ISSUE TITLE
Specified Output Variables missing when running YAML local rules

### ISSUE THREAD

**Issue by ASL-rmarshall** [cdisc-org/cdisc-rules-engine#841]
When running `core validate` with the `-lr` argument pointing to a folder containing YAML rule files, any `Output Variables` specified in the YAML rule files are ignored - only the default output variables (variables referenced in the `Check` specification) are included.

This appears to be caused by an issue in the `load_and_parse_rule` method in `script_utils.py` - `replace_yml_spaces` is being run ***after*** the loaded rule metadata has been passed into `from_cdisc_metadata`, when it should be run ***before*** (because `from_cdisc_metadata` looks for "Output_Variables", not "Output Variables").

The problem is solved by changing:
```python
            if file_extension in [".yml", ".yaml"]:
                loaded_data = Rule.from_cdisc_metadata(yaml.safe_load(file))
                return replace_yml_spaces(loaded_data)
```
to something like:
```python
            if file_extension in [".yml", ".yaml"]:
                loaded_data = yaml.safe_load(file)
                return Rule.from_cdisc_metadata(replace_yml_spaces(loaded_data))
```
I haven't specifically confirmed through testing, but I assume that a similar problem would also occur with any local YAML rule definition that contains `Match Datasets` or a non-default `Rule Type`.
